### PR TITLE
IGAPP-489: Only show timestamp without text for local news

### PR DESCRIPTION
**Pull request by zankerbe**
I modified Page.js so that the text "last update" is shown by default, but can be turned off. Timestamps for local news are now shown without this text.

This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: web/src/modules/common/components/Page.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ type PropsType = {|
   thumbnailSrcSet?: ?string,
   content: string,
   lastUpdate?: Moment,
+  showUpdateText?: boolean,
```
Comment (klinzo) [inline web/src/modules/common/components/Page.js:28]: I'd rename this to `showLastUpdateText` as that is what the component is called, but apart from that nicely done :+1:

**Review by klinzo — APPROVED** [review]
:tada:

**Comment by klinzo** [comment]
Tested on firefox (testumgebung)

File: web/src/modules/common/components/Page.js (hunk 1 above)
Comment (zankerbe) [reply web/src/modules/common/components/Page.js]: Makes sense, i updated it

**Comment by SteffiStuffel** [comment]
Nice job. Tested on Chrome.

Just a little note: normally we name our branches with dashes (-) and not with unterscore (_)

**Review by SteffiStuffel — APPROVED** [review]

**Pull request merged**